src/Components/home/Calendar.jsx

```diff
@@ -25,16 +26,39 @@ const calendarHeader = {
 export function calendarCall(info, successCallback) {
   API.get(`/calendar?start=${info.start}&end=${info.end}`)
     .then((res) => successCallback(res.data))
-    .catch((err) => console.log(err));
+    .catch((err) => console.error(err));
 }
 
 export default function Calendar() {
+
   return (
     <FullCalendar
       defaultView="dayGridMonth"
-      plugins={[dayGridPlugin, timeGridPlugin]}
-      eventSources={[calendarCall, testEvents]}
+      eventTextColor="white"
+      eventBorderColor="rgba(0, 0, 0, 0)"
+      plugins={[dayGridPlugin, timeGridPlugin, googleCalendarPlugin, interactionPlugin]}
+      googleCalendarApiKey='AIzaSyBREtcqxmgY2SbJB5ddscMlKpWrHM0R1JM'
+      events={{googleCalendarId: "q2o6gp4ogd31rls2q7sqvl5nps@group.calendar.google.com"}}
       header={calendarHeader}
+      eventDataTransform={(info) => {
+        info.url = undefined;
+      }}
+      eventRender={(info) => {
+        const {location, description} = info.event.extendedProps;
+        info.el.style.backgroundColor = colors[Math.floor(Math.random() * colors.length)];
```

Comment:
Are we doing a random color just for fun? Would it be worth while to somehow categorize events by color? Can they not come with an event color in the iCal event?

Reply:
FullCalendar doesn't get the color information from Google Calendar by default.  After digging for a bit I didn't see a way to do it.

Yeah, right now I randomly picked a Callibrity color to test out the coloring configuration.  In the future I imagine there will be a key and we'll color based on the title or description.  For example, event with "Anniversary" in the title could be colored Callibrity green.  That would require consistent format in the event entry though.